frontend/src/metabase/writeback/components/ActionViz/ActionForm.tsx

```diff
@@ -21,7 +21,7 @@ interface ActionFormProps {
 function ActionForm({ onSubmit, missingParameters, action }: ActionFormProps) {
   return (
     <DataAppPageFormWrapper>
-      <DataAppPageFormTitle>{action.name}</DataAppPageFormTitle>
+      <DataAppPageFormTitle>{action.name ?? action.slug}</DataAppPageFormTitle>
```

Comment:
Question: seems like we need to do this `name || slug` twist in a few places now. Is it possible to generalize this? Maybe assign a name [the same way we assign IDs](https://github.com/metabase/metabase/blob/038b432275a99219f1bf11a8c8bc44795569beca/frontend/src/metabase/entities/actions/actions.ts#L124) now in the Actions entity?

Reply:
that's a fantastic idea! ❤️